### PR TITLE
Fixes for Remote Execution jobs

### PR DESCRIPTION
**Pull request by ldjebran**
Tests output https://github.com/SatelliteQE/robottelo/pull/6659

### CODE REVIEW

File: airgun/widgets.py [hunk 1]
```diff
@@ -1206,7 +1208,7 @@ def fill(self, value):
     def read(self):
         """Returns string with current widget value"""
         return self.browser.execute_script(
-            "ace.edit('{0}').getValue();".format(self.ace_edit_id))
+            "return ace.edit('{0}').getValue();".format(self.ace_edit_id))
```
Comment (oshtaier) [inline airgun/widgets.py]: does this change fix https://github.com/SatelliteQE/airgun/issues/173 ?

Comment (ldjebran) [reply airgun/widgets.py]: Yes it fix that issue

**Comment by ldjebran** [comment]
@oshtaier comment addressed 
```
(robottelo) dlezz@dlezz:~/projects/robottelo-fork$ pytest -v tests/foreman/ui_airgun/test_jobtemplate.py
============================================ test session starts =============================================
platform linux -- Python 3.6.6, pytest-4.0.2, py-1.5.4, pluggy-0.8.0 -- /home/dlezz/.pyenv/versions/robottelo/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - ON - scope: ffff - storage: redis
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, timeout-1.3.3, services-1.3.1, mock-1.10.0, forked-0.2, cov-2.5.1
collecting ... 2019-01-22 10:59:33 - conftest - DEBUG - BZ deselect is disabled in settings

collected 1 item                                                                                             

tests/foreman/ui_airgun/test_jobtemplate.py::test_positive_end_to_end PASSED                           [100%]

============================================== warnings summary ==============================================
tests/foreman/ui_airgun/test_jobtemplate.py::test_positive_end_to_end
  /home/dlezz/.pyenv/versions/robottelo/lib/python3.6/site-packages/widgetastic/widget/select.py:133: DeprecationWarning: The unescape method is deprecated and will be removed in 3.5, use html.unescape() instead.
    for option
  /home/dlezz/.pyenv/versions/robottelo/lib/python3.6/site-packages/widgetastic/widget/select.py:133: DeprecationWarning: The unescape method is deprecated and will be removed in 3.5, use html.unescape() instead.
    for option
  /home/dlezz/.pyenv/versions/robottelo/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py:796: DeprecationWarning: use driver.switch_to.alert instead
    warnings.warn("use driver.switch_to.alert instead", DeprecationWarning)

================================== 1 passed, 27 warnings in 147.46 seconds ===================================
```

File: airgun/entities/job_template.py [hunk 2]
```diff
@@ -24,10 +24,17 @@ def search(self, value):
...
+    def read(self, entity_name, editor_view_option=None, widget_names=None):
+        """Read Job template values from job template Edit view.
+
+        :param entity_name: Job template name
+        :param editor_view_option: The edit view option to set.
+        :param widget_names: Read only the widgets in widget_names (Optional)
+        """
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
-        return view.read()
+        if editor_view_option is not None:
```
[the remark this answers is not in the document]
Comment (ldjebran) [reply airgun/entities/job_template.py]: resolved in commit 2

**Pull request merged**